book/10-git-internals/sections/objects.asc

```diff
@@ -328,7 +328,7 @@ $ find .git/objects -type f
 
 If you follow all the internal pointers, you get an object graph something like this:
 
-.All the objects in your Git directory.
+.All the reachable objects in your Git directory.
 image::images/data-model-3.png[All the objects in your Git directory.]
```

Comment:
my bad. Should be done now. Added as additional commit. Was wondering how to add the patch as an amend to the first one. Need to do some reading to know how (if it's possible).